src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -22,8 +22,18 @@
                            'Exception will be thrown if free instances exceed one.')
 register_cli_argument('iot hub create', 'unit', help='Units in your IoT Hub.', type=int)
 
+# Arguments for 'iot hub show-connection-string'
+register_cli_argument('iot hub show-connection-string', 'name', options_list=('--name', '-n'),
+                      help='Target IoT Hub name.')
+register_cli_argument('iot hub show-connection-string', 'policy_name',
+                      help='The access policy you choose to use. Default is iothubowner.')
```

Comment:
> Default is iothubowner. [](start = 64, length = 24)

Since you have specified the default in the method signature, it will automatically appear. Right now, it would say "... Default is iothubowner. Default: iothubowner" which is probably not what you want. :)